cmd/ctr/run.go

```diff
@@ -15,23 +15,6 @@ import (
 	"github.com/urfave/cli"
 )
 
-type runConfig struct {
```

Comment:
I included this unrelated fix in the PR just because this is nitpicky.
I can open separate PR if I should.